### DISCUSSION TITLE
Add test id for E2E of the UI

### DISCUSSION THREAD

**Discussion by juancarlosjr97** [directus/directus#13179] [Feature Requests]
Hello,

Sorry if it is repeated, I checked and I could not find another one. 

I was wondering about adding a test id to create E2E for the UI. Thinking about how the project is organized, I consider it an easy task but laborious.

I have E2E written using Cypress, and most of the time they work perfectly and when they do not I just need to tweak them a little bit, but if I could use test-id instead would be awesome!

**Comment by rijkvanzanten** [comment]
What is `test-id`? Is that a lib to include or do you mean just literally add some IDs to elements on the page?

**Reply by juancarlosjr97** [reply]
Yes, add the test id tag so it can be tested, I know it is tricky for the rendering, but was just an idea to allow E2E :) 

FYI: If was not 100% clear https://docs.cypress.io/guides/references/best-practices#How-It-Works

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
